ipatests/test_integration/test_commands.py

```diff
@@ -1561,6 +1561,34 @@ def test_ipa_cacert_manage_prune(self):
 
         assert isrgrootx1_nick in result
 
+    def test_ipa_systemd_journal(self):
+        """
+        This testcase checks that administrative user credentials
+        is not leaked to journald log
+        """
+        cmds = (
+            ['journalctl -t  /usr/sbin/ipa-server-install -n1\
+         -o json-pretty',
+             'journalctl -t  /usr/sbin/ipa-adtrust-install -n1\
+         -o json-pretty',
+             'journalctl -t  /usr/sbin/ipa-kra-install -n1\
+         -o json-pretty',
+             'journalctl -t  /usr/sbin/ipa-restore -n1\
+         -o json-pretty',
+             'journalctl -t  /usr/sbin/ipa-replica-manage -n1\
+         -o json-pretty',
+             'journalctl -t  /usr/sbin/ipa-csreplica-manage -n1\
+         -o json-pretty',
+             'journalctl -t  /usr/sbin/ipa-replica-install -n1\
+         -o json-pretty',
+             'journalctl -t  /usr/sbin/ipa-server-certinstall -n1\
+         -o json-pretty',
+            ]
+            )
+        for cmd in cmds:
+            self.master.run_command(cmds)
+        assert self.master.config.admin_password not in cmds.stdout_text
+        assert self.master.config.dirman_password not in cmds.stdout_text
```

Comment:
these assert statments should be inside the loop.
Also you need to collect the output from the command
```
for cmd in cmds:
    result = self.master.run_command(cmd)
    assert self.master.config.admin_password not in result.stdout_text
    assert self.master.config.admin_password not in result.stdout_text
```